rust-toolchain.toml

```
@@ -0,0 +1,6 @@
+[toolchain]
+versioned = "1.83"
+channel = "stable-x86_64-unknown-linux-gnu"
+components = ["rustfmt", "rustc-dev", "clippy"]
+targets = ["x86_64-unknown-linux-gnu"]
+profile = "minimal"
```

You probably don't want to limit the toolchain to `x86_64-unknown-linux-gnu` since it would prevent target triples like `aarch64-apple-darwin`

```suggestion
[toolchain]
versioned = "1.83"
channel = "stable"
components = ["rustfmt", "rustc-dev", "clippy"]
profile = "minimal"
```